airgun/views/cloud_insights.py

```diff
@@ -3,12 +3,12 @@
 from widgetastic.widget import Text
 from widgetastic.widget import TextInput
 from widgetastic.widget import View
-from widgetastic_patternfly4 import Button
-from widgetastic_patternfly4 import Pagination
+from widgetastic_patternfly4.button import Button
```

Comment:
This doesn't actually change the imported class. 
https://github.com/RedHatQE/widgetastic.patternfly4/blob/master/src/widgetastic_patternfly4/__init__.py#L4

Reply:
yes, it doesn't. I changed it to just make code look good/consistent.

Reply:
I think this is going the wrong way with the consistency.